services/timemanager_service.mjs

```diff
@@ -1,19 +1,42 @@
-import time_registrations from '../fixtures/time_registration.json' assert { type: "json" }
 import config from "config";
 import https from 'https';
 
 class TimeManagerService {
   static async hours_for_project(project) {
     let sum = 0;
     let times = await this.time_registrations_for_project(project)
-    times.forEach(time_registration => {
-      sum += time_registration.billable_hours
-    })
+
+    await Promise.all(times.map(async time_registration => {
+      sum += await time_registration.billable_hours
+    }))
```

Comment:
all of this async stuff can go away, calling `time_registration.billable_hours` isn't async